control/nlsys.py

```diff
@@ -1317,7 +1317,7 @@ def nlsys(
 
 
 def input_output_response(
-        sys, T, U=0., X0=0, params=None, ignore_errors=False,
+        sysdata, T, U=0., X0=0, params=None, ignore_errors=False,
```

Comment:
Changing the parameter name `sys` to `sysdata` here and in other functions modified by this pull request is good for clarity to users, but it will break existing calls of the form `input_output_response(sys=ss, ...)`. As such, the next version probably should be 0.11.0 instead of 0.10.1 (https://github.com/python-control/python-control/issues/1020) to emphasize that existing code using `control` may need to be updated.